particle/particle/particle.py

```diff
@@ -378,8 +385,12 @@ def load_table(cls, filename=None, append=False):
             cls._table_names = []
 
         if filename is None:
-            filename = data.open_text(data, "particle2019.csv")
-            cls._table_names.append("particle2019.csv")
+            import fileinput
```

Comment:
This should be imported at the top - importing inside a function should only be done for special cases, a standard library module is just fine at the top.

Reply:
OK, will get to this and the rest tomorrow. Let me call it a day - it's getting rather late.

Thanks for starting to take a look, BTW.

Reply:
Done putting the imports at the start of the file.